lighthouse-cli/test/smokehouse/report-assert.js

```diff
@@ -427,19 +425,20 @@ function reportAssertion(localConsole, assertion) {
           log.greenify(assertion.actual));
     }
   } else {
-    if (assertion.diff) {
-      const diff = assertion.diff;
-      const fullActual = String(JSON.stringify(assertion.actual, null, 2))
```

Comment:
little did you know this was a [load-bearing `String()`](https://github.com/GoogleChrome/lighthouse/pull/9130/files#r290982686) (try renaming `LinkElements` to `LinkElementz` or whatever :)

Though I certainly didn't remember why it was there, so we can change it to something more obvious that does the same thing

Reply:
Made it more explicit and added tests.